source/module_hsolver/hsolver_lcao.h

```diff
@@ -41,6 +41,8 @@ class HSolverLCAO : public HSolver<T, Device>
     );*/
     const Parallel_Orbitals* ParaV;
 
+    void parakSolve(hamilt::Hamilt<T>* pHamilt, psi::Psi<T>& psi, elecstate::ElecState* pes);
```

Comment:
What is this function used for?

Reply:
This is exactly the function used to diagonalize Hk with k-points parallelism in HsolverLCAO.